doc/source/conf.py

```diff
@@ -76,6 +76,7 @@
     # to ensure that include files (partial pages) aren't built, exclude them
     # https://github.com/sphinx-doc/sphinx/issues/1965#issuecomment-124732907
     "**/includes/**",
+    "**/api/pandas.Series.dt.rst",
```

Comment:
Just a reminder to open an issue about this once we merge this

Reply:
Yep, it's on my todo list